### ISSUE TITLE
ebay-progress-stepper: wrong html generated

### ISSUE THREAD

**Issue by darkwebdev** [eBay/ebayui-core#1367]
# Bug Report

## Description
The component always wraps the step text with `p` making it impossible to match a vertical Skin example (Skin 12 documentation screenshot):
<img width="293" alt="image" src="https://user-images.githubusercontent.com/2737310/113886749-36466f00-97c1-11eb-9149-c65ec27269ae.png">

Component usage example from the [storybook](https://ebay.github.io/ebayui-core/?path=/story/ebay-progress-stepper-direction--vertical):
```
<@step>
    <@title>Order placed</@title>
    New Mens Addidas Ultra Boost<br/>
    Order total $220
</@step>
```

HTML produced by component:
<img width="291" alt="image" src="https://user-images.githubusercontent.com/2737310/113886958-5d04a580-97c1-11eb-92ea-ea7b6665539a.png">

Possible solutions:

1. Not wrap user's content in `p` and ask users to use `p`themselves:
```
<@step>
    <@title>Order placed</@title>
    <p>New Mens Addidas Ultra Boost</p>
    <p>Order total $220</p>
</@step>
```

2. Introduce a child component (for example `@text` or `@body`) and use it like:
```
<@step>
    <@title>Order placed</@title>
    <@text>New Mens Addidas Ultra Boost</@text>
    <@text>Order total $220</@text>
</@step>
```

3. Wrap user's content in `p` only if it's not already a `p`

**Comment by austingardner** [comment]
Personally I'm a fan of approach #1... I'll put up a PR

**Comment by ianmcburnie** [comment]
Addressed in 7.4.0
